### ISSUE TITLE
test 10

### ISSUE THREAD

**Issue by mickogoin** [mickogoin/test_repo#10]
![image](https://user-images.githubusercontent.com/79158859/215924272-cdaebf0d-6a76-4478-adef-8459a300f545.png)

![image](https://user-images.githubusercontent.com/79158859/161542972-dd55bc27-4b5a-4854-9f9c-31818e63c43b.png)

![image](https://user-images.githubusercontent.com/79158859/161542984-c2bfb4db-7e30-45c0-87f4-4a705a9b4fb1.png)

![image](https://user-images.githubusercontent.com/79158859/161543012-3f68c3be-440c-4c80-885b-8fe02baaa11e.png)

![image](https://user-images.githubusercontent.com/79158859/161543027-875cd724-5000-4907-82cd-1771e961671d.png)

Romulo Paano [planned trip](https://radar.goin.org/riders/44d9ff1a-9fb1-4800-adc6-f63b5d3b5d91/profile/recurringTrips) vs [set trip](https://radar.goin.org/trips-management)

Robert Bodeen [planned trip](https://radar.goin.org/riders/d33591d4-4521-447a-931b-2becf922e46c/profile/recurringTrips) vs [set trip](https://radar.goin.org/trips-management)

Wayne Radder [planned trip](https://radar.goin.org/riders/47e6533c-5f04-4dab-91b3-4298efc306f9/profile/recurringTrips) vs [set trip](https://radar.goin.org/trips-management)

Basically all the trips that I updated have this error

**Comment by mickogoin** [comment]
![image](https://user-images.githubusercontent.com/79158859/160435429-796baff4-abf8-4865-b230-f0fa06d6622b.png)
